src/apiProxy/apiConfigCreators.js

```diff
@@ -30,6 +30,34 @@ export function group(params) {
 }
 
 /**
+ * group communication endpoints
+ *
+ * accepts an `method` property in params
+ */
+export function mugcomm(params) {
+	const path = 'communications';
+	const { urlname, conversationId } = params;
```

Comment:
Loving all the functional styles I see in the new platform; this is an elegant way to handle creating a path based on which values are actually populated from `params`.

One concern I have is that we would lose the self-documenting aspect of the `method` values in the `switch` statement. The group communications v0 endpoint for example, is not documented in the public API yet. I suspect we'll run into this with a number of new features...

I could always list accepted values in a comment above, but compared to actual code, a comment is sometimes a lie waiting to happen.